certified-connectors/U.S. Bank Treasury Management/readme.md

```diff
@@ -0,0 +1,59 @@
+## U.S. Bank 
```

Comment:
@jzworkman this great readme file. is it possible to include all required sections from this [template](https://github.com/microsoft/PowerPlatformConnectors/blob/dev/templates/certified-connectors/readme.md). you can see the code [here](https://raw.githubusercontent.com/microsoft/PowerPlatformConnectors/dev/templates/certified-connectors/readme.md).